coc/hero.py

```diff
@@ -13,43 +14,45 @@
 
 
 class Hero(DataContainer):
-    """Represents a Hero object as returned by the API, optionally filled with game data.
+    """
+    Represents a Hero object as returned by the API, optionally
+    filled with game data.
 
     Attributes
     ----------
-    id: int
+    id: :class:`int`
```

Comment:
is this required for sphinx? I remember going from `:class:int` to `int`, so interested to know!